pyproject.toml

```diff
@@ -0,0 +1,145 @@
+# Copyright (C) British Crown (Met Office) & Contributors.
+#
+# This file is part of Rose, a framework for meteorological suites.
+#
+# Rose is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Rose is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Rose. If not, see <http://www.gnu.org/licenses/>.
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "metomi-rose"
+description = "Rose, a framework for meteorological suites."
+authors = [
+    { name = "British Crown (Met Office) & Contributors", email = "metomi@metoffice.gov.uk" }
+]
+readme = "README.md"
+license = { text = "GPL" }
+keywords = ["hpc", "weather-modelling", "weather-model", "meteorological-suites", "meteorological-modelling", "meteorological-models"]
```

Comment:
Only semi relevent, but we should consider adding "workflows" and "workflow automation" to this list.